src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -177,16 +176,19 @@ const Map: React.FC<Props> = ({ setSidebarContent, setLocateFunction, setPageTit
       get(`/api/${identifier}`).then((response) => {
         setLandscape(response);
         setPageTitle(response.name);
-        if (locateComponent) {
-          setLocateFunction(() => locateComponent);
-        }
       });
 
       get(`/assessment/${identifier}`).then((response) => {
         setAssessments(response);
       });
     }
-  }, [identifier, setPageTitle, setLocateFunction, locateComponent, landscape]);
+  }, [identifier, setPageTitle, landscape]);
+
+  useEffect(() => {
+    if (locateComponent) {
+      setLocateFunction(() => locateComponent);
+    }
+  }, [value, setLocateFunction, locateComponent]);
```

Comment:
Just as a question, are these all deps that are needed?

Reply:
At least `value` is not needed in line 191.

Reply:
fixed